Dockerfile

```diff
@@ -35,7 +35,7 @@ USER docsworker-xlarge
 WORKDIR /home/docsworker-xlarge
 
 # get shared.mk
-RUN curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/makefiles/shared.mk -o shared.mk
+RUN curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/DOP-2357/makefiles/shared.mk -o shared.mk
```

Comment:
Probably not immediately actionable, but shared.mk should probably be pulled when we pull the entrypoint makefile, not during image creation

Reply:
Also, is it desirable/possible to parameterize where we pull makefiles from?

Reply:
sure

Reply:
We will address these as part of our rearchitecture project